src/scss/components/card/faux-cards/_faux-card--layout--media-text-area.scss

```diff
@@ -7,26 +7,24 @@
       .paragraph-item.ptype-stanford-wysiwyg.flex-6-of-12 ~ .paragraph-item.ptype-stanford-media-caption.flex-6-of-12,
       // set margin and spacing for media caption on the left/first
       .paragraph-item.ptype-stanford-media-caption.flex-6-of-12 ~ .paragraph-item.ptype-stanford-wysiwyg.flex-6-of-12 {
-        @include grid-media-max(md) {
+        @include grid-media-max('md') {
           margin-top: -$mobile-modular-spacing-2; // negative margin required in this one case, to counteract the previous element's bottom margin, position doesn't work as ewll
           margin-bottom: $mobile-modular-spacing-7;
         }
-        @include grid-media-min(md) {
-          @include grid-media-max(lg) {
-            margin-top: -90px; // to counteract the previous element's bottom margin,
-            @include modular-spacing('margin-bottom', 5);
-          }
+        @include grid-media-between('md', 'lg') {
+          margin-top: -90px; // to counteract the previous element's bottom margin,
+          @include modular-spacing('margin-bottom', 5);
         }
 
         // positioning is neccessary since we can only style this unique case by styling the second element
-        @include grid-media-min(lg) { // 992
+        @include grid-media-min('lg') {
           max-width: 44%;
           margin-top: 0;
           margin-left: 0;
           margin-right: 3%;
```

Comment:
FYI we have a mixin so you could also do `@include margin(0 3% null 0);` to save some typing but this is entirely optional 😃

Reply:
oh i didn't know this was an option.